jerry-core/ecma/operations/ecma-proxy-object.c

```diff
@@ -422,6 +422,11 @@ ecma_proxy_object_set_prototype_of (ecma_object_t *obj_p, /**< proxy object */
 
   ecma_free_value (trap_result);
 
+  if (!boolean_trap_result)
```

Comment:
Is this from a newer standard? If so it would be great if we could reference the standard, maybe even update all of the other references for the standards (for example see the proxy_create method above).